src/components/Icon/index.js

```diff
@@ -48,7 +52,9 @@ class Icon extends PureComponent {
                     accessibilityHint={`${this.props.src.name} Icon`}
                     style={[StyleUtils.getWidthAndHeightStyle(width, height), styles.bgTransparent, styles.overflowVisible]}
                 >
-                    <View style={[StyleUtils.getWidthAndHeightStyle(width, height), IconWrapperStyles, styles.pAbsolute]}>
+                    <View style={[StyleUtils.getWidthAndHeightStyle(width, height), IconWrapperStyles, styles.pAbsolute,
+                        StyleUtils.getAvatarSVGBorder(this.props.svgAvatar)]}
+                    >
```

Comment:
NAB; Since this is getting pretty long, I would slightly prefer capturing all of this in a `const` variable out of the JSX, then plopping that variable in here. If you disagree feel free to disregard :D